src/app/search/components/results/results.component.tsx

```diff
@@ -54,7 +66,7 @@ export default function ResultsComponent() {
                 <p>{Math.round(doctor.averageRating * 100) / 100}</p>
               </div>
             </div>
-            <button>دریافت نوبت</button>
+            <Link href={``}>دریافت نوبت</Link>
```

Comment:
زمانی که یه لینک دارید که فعلاً قرار نیست کار خاصی انجام بده، بهتره از هشتگ استفاده کنید

```suggestion
            <Link href="#">دریافت نوبت</Link>
```